draft-ietf-mls-protocol.md

```diff
@@ -1063,11 +1065,13 @@ it for any further processing.
 uint64 expiration;
 ~~~~~
 
-Note that as an extension, it is not required that any given ClientInitKey have
-an expiration time.  In particular, applications that rely on "last resort"
-ClientInitKeys to ensure continued reachability may choose to omit the
-expiration extension from these keys, or give them much longer lifetimes than
-other ClientInitKeys.
+Applications that rely on "last resort" ClientInitKeys MAY set the
+expiration to its maximum value even though this is NOT RECOMMENDED.
+It is RECOMMENDED to rotate last resort keys at a pace chosen by the
+application even though they can have much longer lifetimes than other
+ClientInitKeys.
+
+This extension MUST be always present in an MLS ClientInitKey.
```

Comment:
The need for this is not clear to me.  Let's discuss this at the interim before merging this PR.

Reply:
This was discussed at the interim, and people were comfortable with this requirement.